osconfig_tests/main.go

```diff
@@ -85,23 +81,25 @@ func main() {
 		close(tests)
 	}()
 
-	var testSuites []*junitxml.TestSuite
 	for ret := range tests {
-		testSuites = append(testSuites, ret)
-	}
+		testSuiteOutPath := filepath.Join(*outDir, fmt.Sprintf("%s.xml", ret.Name))
```

Comment:
For clarity:
fmt.Sprintf("%s_junit.xml", ret.Name)

Reply:
done